src/main/resources/application.properties

```diff
@@ -1,6 +1 @@
 server.port=8080
-spring.data.mongodb.uri=mongodb+srv://grupo3-aerogenious:bkediB84oNGxTsTj@cluster0.5pvb5dz.mongodb.net/Flightwise
-spring.data.mongodb.database=Flightwise
-
```

Comment:
You should maintain these lines, but just anonymize the values.

Something like spring.data.mongodb.uri=DATABASE-URI.

It indicates what kind of value it expects, but doesn't show them.